index.html

```diff
@@ -0,0 +1,100 @@
+        rel="stylesheet">
+</head>
+
+<body>
+    <header>
+        <div class="login">
+            <a href="/"><img src="/misson_1/img/sm.png" alt="logo"></a>
+            <a href="/login.html" class="logo_bt">로그인</a>
+        </div>
+    </header>
+    <!-- pc -->
+    <section class="pc">
+        <div class="main">
+            <div class="main_all">
+                <div class="items_title">
+                    <span>일상의 모든 물건을<br> 거래해 보세요</span>
+                    <a class="items" href="/items.html">구경하러 가기</a>
+                </div>
+                <img src="/misson_1/img/pc/header.png" alt="mainheader">
+            </div>
+        </div>
+    </section>
+    <section class="pc">
+        <div class="hotitem">
+            <img src="/misson_1/img/pc/sec1.png" alt="hotitemimg">
+            <div class="hotitem_text">
+                <span class="tx_01">Hot item</span>
+                <span class="tx_02">인기 상품을<br>확인해 보세요</span>
+                <span class="tx_03">가장 HOT한 중고거래 물품을<br>판다 마켓에서 확인해 보세요</span>
+            </div>
+        </div>
+    </section>
+    <section class="pc">
+        <div class="search">
+            <div class="search_text">
+                <span class="tx_01">Search</span>
+                <span class="tx_02">구매를 원하는<br>상품을 검색하세요</span>
+                <span class="tx_03">구매하고 싶은 물품은 검색해서<br>쉽게 찾아보세요</span>
+            </div>
+            <img src="/misson_1/img/pc/sec2.png" alt="searchimg">
+        </div>
+    </section>
+    <section class="pc">
+        <div class="box">
+            <div class="register">
+                <img src="/misson_1/img/pc/sec3.png" alt="registerimg">
+                <div class="register_text">
+                    <span class="tx_01">Register</span>
+                    <span class="tx_02">판매를 원하는<br>상품을 등록하세요</span>
+                    <span class="tx_03">어떤 물건이든 판매하고 싶은 물품을<br>판다 마켓에서 확인해 보세요</span>
+                </div>
+            </div>
+        </div>
+    </section>
+    <section class="pc">
+        <div class="footerimg">
+            <div class="footer_all">
+                <div class="footer_title">
+                    <span>믿을 수 있는<br>판다마켓 중고 거래</span>
+                </div>
+                <img src="/misson_1/img/pc/footer.png" alt="mainheader">
+            </div>
+        </div>
+    </section>
+    <!-- pc -->
+    <!-- mobile -->
+
+    <!-- mobile -->
+    <footer>
+        <div class="footer_intro">
+            <span class="address">@codeit - 2025</span>
+            <div class="link">
+                <a class="link_tx" href="/privacy">Privacy Policy</a>
+                <a class="link_tx" href="/faq">FAQ</a>
+            </div>
+            <div class="sns">
+                <a href="https://www.facebook.com/PandaMarket/" target="_blank"><img src="/misson_1/img/facebook.png"
+                        alt=""></a>
```

Comment:
💊 제안  
이 영역은 로고들의 모음이라고 할 수 있습니다. 관련된 요소들끼리 묶인 목록이니 의미록적인 마크업을 작성한다면 list 관련 태그로 작성하시는 것을 추천드립니다.
또한 태그별 위계관계가 잘 보이도록 작성해주세요!
```suggestion
            <ul class="sns">
                <li>
                  <a href="https://www.facebook.com/PandaMarket/" target="_blank">
                    <img src="/misson_1/img/facebook.png" alt="">
                  </a>
                </li>
```